packages/python/stitch/.github/workflows/build.yml

```diff
@@ -0,0 +1,44 @@
+name: Build
+
+on:
+  push:
+    branches: main
+  pull_request:
+    branches: "*"
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        python-version: ["3.7", "3.10"]
```

Comment:
Python 3.7??????

Reply:
The ipywidget (jupyter widgets) developers recommend building custom widgets via a cookie-cutter template they provide (https://github.com/jupyter-widgets/ipywidgets) to follow best practice for interactive widgets.

Looks like this is part of the template (https://github.com/jupyter-widgets/widget-ts-cookiecutter/blob/master/.github/workflows/build.yml).

I'm in mind that we delete this. We can also update it and hook it to the CI if needed. I'm not fussed either way.

Reply:
Related note: hopefully there's a way we can hook playwright and backstopJS for clientside testing.